sdk/search/Azure.Search.Documents/src/Generated/Models/SearchIndexer.Serialization.cs

```diff
@@ -34,13 +34,27 @@ void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
             writer.WriteStringValue(TargetIndexName);
             if (Optional.IsDefined(Schedule))
             {
-                writer.WritePropertyName("schedule");
-                writer.WriteObjectValue(Schedule);
+                if (Schedule != null)
```

Comment:
Not to hold up this PR, but would it make sense to either have a method on `Optional<T>` to write to a `Utf8JsonWriter` or an extension method (even if internal) that would just do this, rather than repeat this code over and over again? Makes it central to change as well.

Reply:
It's a good idea, the current implementation of the generator makes it a little bit hard but it's definitely something to look at.